src/components/pages/GarmentDetail.vue

```diff
@@ -90,44 +90,93 @@
       CommitInfo,
       Loader,
     },
-    // beforeCreate() {
-    //   const id = this.$route.params.id;
-
-    //   axios.get(`http://localhost:3000/garments/${id}`)
-    //     .then((response) => {
-    //       this.garment = response.data;
-    //       this.dataIsLoaded = true;
-    //     })
-    //     .catch(error => this.showError(error.message));
-    // },
     data() {
       return {
-        garment: {},
+        garment: {
+          reference: '',
+          title: '',
+          description: '',
+          status: '',
+          type: '',
+          creator: '',
+          creation_date: '',
+          licence: 'Information not available yet on public API',
+          commitChanges: '',
+          contributors: '',
+          numberOfProposals: '',
```

Comment:
commitChanges and numberOfProposals are numbers? if so, they should default to a number